Vendors/tropter/cmake/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ if(TROPTER_COPY_DEPENDENCIES AND APPLE)
             ${gfortran}
             ${quadmath}
             ${gcc_libdir}/libgcc_s.1.dylib
+            ${gcc_libdir}/libgcc_s.1.1.dylib
 
```

Comment:
Can we introduce a cmake variable with the name of the dylib for future maintenance so in case it changes in the future we're not digging through the complicated mess you had to go thru this time around @AlbertoCasasOrtiz

Reply:
I have created an issue with the cmake variable change.